resources/monitoring/templates/grafana/dashboards-1.14/pods.yaml

```diff
@@ -1,678 +0,0 @@
-{{- /*
-Generated from 'pods' from https://raw.githubusercontent.com/coreos/kube-prometheus/master/manifests/grafana-dashboardDefinitions.yaml
```

Comment:
This is one of the most used dashboards. We should check what replaced it and bring it back if there is no replacement.

Reply:
Hm, i guess they got rid of it since it overlaps with Kubernetes / Compute Resources / Pod and Kubernetes / Networking / Pod. Neither of those 2 visualise the container restart metric though.

Reply:
As per discussion with @a-thaler decided to keep it as a Kyma dashboard